blocks/byline-block/themes/news.json

```diff
@@ -4,12 +4,16 @@
 			"default": {
 				"font-size": "var(--body-font-weight)",
 				"line-height": "var(--body-line-height)",
+				"color": "var(--global-neutral-8)",
+				"text-decoration": "none",
 				"components": {
 					"link": {
-						"text-decoration": "none"
+						"color": "var(--global-neutral-8)",
```

Comment:
Small change, but instead of using the global tokens for these text colors, can we use the alias tokens instead? It would just be `text-color` instead of `global-neutral-8`, and `text-color-subtle` instead of `global-neutral-5`